integration-tests/src/mock_network/setup.rs

```diff
@@ -125,7 +125,7 @@ impl FromStr for MockNetworkMode {
 pub fn setup_mock_network(
     client_home_dir: &Path,
     network_home_dir: &Path,
-    config: &NearConfig,
+    config: NearConfig,
```

Comment:
Asking to learn: any specific reason why we remove the borrowing?

Reply:
It's because the line I added below requires cloning the config if I use the reference. I removed the borrowing to save the clone.